tdvt/README.md

```diff
@@ -0,0 +1,72 @@
+# TDVT datasets
+
+## Import the datasets in your cluster
+### Calcs
+```
+mongoimport --uri "{your connection URL}" --db=tdvt --drop --collection Calcs --type=json --file={Path to tdvt/datasets directory}/Calcs.json
+```
+### Staples
+```
+mongoimport --uri "{your connection URL}" --db=tdvt --drop --collection Staples --type=json --file={Path to tdvt/datasets directory}/Staples.json
+```
+
+## Configure your Data Federation
+Add the following to `databases`
```

Comment:
Let's add a bit more detail to this line: such as for the local ADF the configuration change is in the file `./testdata/config/mongodb_local/tenant-config.json`.  For remote ADF ensure the database and collection names match the below configuration.